src/web/modules/transactions/refundConfirmation.njk

```diff
@@ -0,0 +1,49 @@
+{% from "common/json.njk" import json %}
+
+{% extends "layout/layout.njk" %}
+
+{% from "govuk/components/input/macro.njk" import govukInput %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+
+{% set isTestData = not (parentTransaction and parentTransaction.live) %}
+
+{% block main %}
+  <h1 class="govuk-heading-m">Refund</h1>
+
+  <div class="govuk-body govuk-!-margin-bottom-4">
+    <a href="/transactions/{{ transaction.transaction_id }}" class="govuk-back-link">Refunds
+      payment {{ transaction.transaction_id }}</a>
+  </div>
+
+  <div class="status-inline-spacer">
+    <span class="govuk-body-l status-amount">{{ (transaction.total_amount or transaction.amount) | currency }}</span>
+  </div>
+
+  <div class="govuk-grid-row status-spacer payment-detail-row">
+    <div class="govuk-grid-column-one-quarter">
+      <span class="govuk-caption-m">Date</span>
+      <span class="govuk-body">{{ transaction.created_date | formatDate }}</span>
+    </div>
+  </div>
+
+  <h1 class="govuk-heading-m">Confirm refund</h1>
+
+  <form method="post" action="/fix-failed-stripe-refund/:id">
+
```

Comment:
We should have some text here explaining what this does. We can add that later. For now, put a warning here that this is under development and nobody should press the button yet. You can use the [warning text](https://design-system.service.gov.uk/components/warning-text/) component!